frontend/public/components/resource-quota.jsx

```diff
@@ -57,17 +71,35 @@ export const getQuotaResourceTypes = (quota) => {
   return _.keys(specHard).sort();
 };
 
-const getResourceUsage = (quota, resourceType) => {
+const getResourceUsage = (quota, resourceType, namespace, isACRQ = false) => {
   const isCluster = isClusterQuota(quota);
   const statusPath = isCluster ? ['status', 'total', 'hard'] : ['status', 'hard'];
   const specPath = isCluster ? ['spec', 'quota', 'hard'] : ['spec', 'hard'];
   const usedPath = isCluster ? ['status', 'total', 'used'] : ['status', 'used'];
+  let totalUsed;
+  let used;
+
+  // ACRQ is a cluster-scoped resource that behaves like a namespaced resource
+  // It has cluster-scoped data as well as namespaced data
+  // "used" is the data for the current namespace and "totalUsed" is the cluster-scoped data
+  if (isACRQ) {
+    used = _.get(quota, ['status', 'namespaces']);
```

Comment:
Don't reuse the `used` variables to mean different things. Assign each temp value to a different variable.